articles/dns/dns-faq-private.yml

```
@@ -102,6 +102,12 @@ sections:
           How do I migrate my existing private DNS zones to the new model?
         answer: |
           We strongly recommend that you migrate to the new resource model as soon as possible. Legacy resource model will be supported, however, further features won't be developed on top of this model. In future, we intend to deprecate it in favor of new resource model. For guidance on how to migrate your existing private DNS zones to new resource model see[migration guide for Azure DNS private zones](private-dns-migration-guide.md).
+
+      - question: |
+          Which one takes precedence, Azure private DNS or a custom DNS server setup on a VM?
+        answer: |
+          That depends how you configured your Azure VMs/VNet. If you specify custom DNS at the VNet or NIC level (by pointing to the IP address of the NIC of the VM running the DNS server), this will take precedence. If not, then the private DNS zone linked to the VNet will apply
```

```suggestion
          That depends how you configured your Azure VMs or Azure private DNS. If you specify custom DNS at the virtual-network or NIC level (by pointing to the IP address of the NIC of the VM running the DNS server), this will take precedence. If not, then the private DNS zone linked to the virtual network will apply.
```
Edits for Microsoft style